style.css

```diff
@@ -1,818 +1,668 @@
+
+.program-container {
+  background-image: url('static/pjbck.jpg');
+  background-size: cover;
+  color: #fff;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  padding: 0;
-  padding-left: 50px;
-  padding-right: 50px;
-  height: 289px;
-  width: 289px;
-  margin: 9px auto;
+  align-items: center;
 }
 
-.lan {
-  background: #1c528c;
-  background-image: url("./images/lfs/hast.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
+.pr-section-title {
+  margin-top: 4%;
 }
 
-.fra {
-  background-color: #e2505c;
-  background-image: url("images/lfs/cuadrado.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
-  background-size: 33%;
+.pr-section-title::after {
+  content: "";
+  display: block;
+  width: 45px;
+  margin-left: 55px;
+  padding-top: 10px;
+  opacity: 0.7;
+  border-bottom: 3px solid #ec5242;
 }
 
-.ski {
-  background-color: #febf52;
-  background-image: url("images/lfs/foco.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
-  background-size: 35%;
+.card-container {
+  display: flex;
+  max-width: 75%;
+  gap: 3px;
+  margin-top: 4%;
 }
 
-h4 {
-  font-size: 20px;
-  font-weight: 800;
-  color: #fff;
+.card {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+  align-items: center;
+  text-align: center;
+  background-color: rgba(100, 100, 100, 0.5);
 }
 
-.inBox {
-  color: #fff;
-  list-style-position: inside;
-  justify-self: center;
+.card-img {
+  min-height: 50px;
+  min-width: 50px;
+  margin: 10px 0 5px 0;
 }
 
-.image-about {
-  display: none;
+.card-title {
+  color: #ec5242;
+  font-size: 22px;
+  margin: 0;
 }
 
-button {
-  transition-duration: 0.4s;
-  cursor: pointer;
+.card-text {
+  font-size: 13px;
+  margin: 0;
+  max-width: 85%;
+  margin-bottom: 7%;
 }
 
-button:hover {
-  background: #fff;
-  border-color: black;
-  border-width: 1px;
-  color: #1a2236;
+.projects-link {
+  background-color: transparent;
+  color: #fff;
+  text-decoration: underline;
+  margin-top: 4%;
+  margin-bottom: 4%;
 }
 
-button:active {
-  border-width: 4px;
+.projects-link-mobile {
+  display: none;
 }
 
-/* formulary section */
-
-.formulary {
+.spk-container {
   display: flex;
   flex-direction: column;
-  background: #f9f9fa;
-  background-image: url(images/image_geometry_1.png);
-  background-repeat: no-repeat;
-  padding: 1px;
+  padding-bottom: 14%;
 }
 
-.space {
-  height: 106px;
-
-  /* border: #1a2236 dotted 1px; */
+.spk-container-title {
+  align-self: center;
+  margin-top: 4%;
+  margin-bottom: 4%;
 }
 
-.cont-form {
-  width: 320px;
+.spk-container-title::after {
+  content: "";
+  display: block;
+  width: 45px;
+  margin-left: 75px;
+  padding-top: 10px;
+  opacity: 0.7;
+  border-bottom: 3px solid #ec5242;
+}
 
-  /* border: #1a2236 dotted 1px; */
+.spk-card-container {
   align-self: center;
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 10%;
+  width: 90%;
 }
```

Comment:
Kindly ensure that the speakers section is as close to the design as possible. Note the following:

- Please, add more padding of the section to match the design
- The orange underline of the heading ("GUESTS") of the section should be centralised

Your design:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/bd290a8d-7da3-40a2-a09e-fd690822be19)


Design Guide:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/5f9353d4-639f-4305-a3ec-167d6c5c7c84)